health/micro-ui/web/packages/modules/campaign-manager/src/pages/employee/NewAppConfiguration/NewDependentFieldWrapper.js

```
@@ -0,0 +1,1085 @@
+                stroke={true}
+                style={{
+                    background: "#EFF8FF",
+                    height: "fit-content",
+                }}
+                className={"version-tag"}
+                labelStyle={{ whiteSpace: "normal", wordBreak: "break-word" }}
+            />
+
+            <div style={{ display: "inline-flex", alignItems: "center", gap: "0.75rem", whiteSpace: "nowrap" }}>
+                <div
+                    role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)